bindings/nodejs/generated.d.ts

```diff
@@ -444,49 +401,41 @@ export class Operator {
   /**
    * List given path.
    *
-   * This function will create a new handle to list entries.
+   * This function will return an array of entries.
    *
    * An error will be returned if given path doesn't end with `/`.
    *
    * ### Example
    * ```javascript
-   * const lister = await op.list("path/to/dir/");
-   * while (true) {
-   *   const entry = await lister.next();
-   *   if (entry === null) {
-   *     break;
-   *   }
+   * const list = await op.list("path/to/dir/");
+   * for (let entry of list) {
    *   let meta = await op.stat(entry.path);
    *   if (meta.isFile) {
    *     // do something
    *   }
    * }
    * ```
    */
-  list(path: string): Promise<Lister>
+  list(path: string, options?: ListOptions | undefined | null): Promise<Array<Entry>>
```

Comment:
Would you like to give an example that uses list with recursive?

Reply:
Updated